index.html

```diff
@@ -153,6 +153,7 @@ <h2>
           readonly attribute GamepadMappingType mapping;
           readonly attribute FrozenArray&lt;double&gt; axes;
           readonly attribute FrozenArray&lt;GamepadButton&gt; buttons;
+          readonly attribute FrozenArray&lt;GamepadTouch&gt;? touches;
```

Comment:
nit: Remove since it won't be nullable?
```suggestion
          readonly attribute FrozenArray&lt;GamepadTouch&gt; touches;
```

Reply:
Done